### DISCUSSION TITLE
Why was MRTCore’s namespace changed?

### DISCUSSION THREAD

**Discussion by wjk** [microsoft/WindowsAppSDK#951] [Q&A] [answered]
What the title says, really. As seen in https://github.com/microsoft/ProjectReunion/pull/948.

**Comment by andrewleader** [comment, marked as answer]
Hey William, we changed this API to be consistent with other Windows App SDK APIs, which will all be under `Microsoft.Windows.*` (since they're all Windows APIs and not Microsoft APIs).

**Reply by wjk** [reply]
Will WinUI 3 be updated to match, or will that be staying at `Microsoft.UI.Xaml`?

**Reply by andrewleader** [reply]
WinUI 3 will likely be staying with `Microsoft.UI.*` simply due to the amount of code and avoiding causing migration pains for developers currently using it - but open to feedback on if you or others think it should be moved under `Microsoft.Windows.*`!

**Reply by wjk** [reply]
I agree that WinUI’s namespace should stay put.
